### ISSUE TITLE
Templated content creation fails if any of the node observer types is missing

### ISSUE THREAD

**Issue by tusmester** [SenseNet/sensenet#135]
When you create a content from a template that contains a large structure (e.g. a workspace) and there are local reference values that need to be updated during templated creation, we try to load a couple of node observer types to **disable them during save**. If any of them is missing, we throw an exception, which is incorrect, because it is not an error if a node observer is not present because it is _deployed in another component_.

**Solution**: load node observer types gracefully, without throwing an exception.

```
TypeNotFoundException: Type was not found: SenseNet.Workflow.WorkflowNotificationObserver
   at SenseNet.Tools.TypeResolver.FindTypeInAppDomain(String typeName, Boolean throwOnError)
   at SenseNet.Tools.TypeResolver.GetType(String typeName, Boolean throwOnError)
   at SenseNet.ContentRepository.ContentTemplate.UpdateLocalReferences(Node templateRoot, Node targetRoot) 
in C:\Dev\github\sensenet\src\ContentRepository\ContentTemplate.cs:line 529
```

**Comment by tusmester** [comment]
It is possible the https://github.com/SenseNet/sn-workspaces/issues/3 is also related to this one.

**Comment by tusmester** [comment]
Fixed by #136
